doc/workshop/building.rst

```diff
@@ -8,55 +8,43 @@ Requirements
 ------------
 
 - Install packer (http://packer.io/)
-- Clone the packer-templates repository
-  (https://github.com/kaorimatz/packer-templates)
+- Install Vagrant, libvirt and VirtualBox
+- Clone the Fedora kickstarts repo (https://pagure.io/fedora-kickstarts)
 
 
 Packer template
 ---------------
 
-Apply the following changes to the ``fedora-28-x86_64.json`` packer
-template:
+Packer template ``packer-template-fedora.json`` requires Fedora 34 kickstart file
+used by Fedora to build vagrant images:
 
-- Add the ``scripts/fedora/ipa.sh`` provisioner and copy (or
-  symlink) ``ipa.sh`` from *this* repository to ``scripts/fedora``.
-  This script installs the FreeIPA packages and creates other files
-  required for the workshop.
+- Clone the repo and checkout latest Fedora release branch::
 
+  $ git clone https://pagure.io/fedora-kickstarts.git
+  $ git checkout f34
```

Comment:
The step "cd fedora-kickstarts" is missing before git checkout

Reply:
Fixed.